docs/3.0.0-beta.x/getting-started/contributing.md

```diff
@@ -0,0 +1,21 @@
+# 🦸 Contributing
+
+Strapi is a community oriented project and we really appreciates every contribution made by the community: feature requests, bug reports, and especially pull requests! If you have any questions please reach out to our Core team on [Slack](https://slack.strapi.io).
```

Comment:
I'm not sure having a list of every `@...` is helpful.
If the person check [main contributors](https://github.com/strapi/strapi/graphs/contributors) of the project or the [company page](https://strapi.io/company) you can easily find us on Slack.

Reply:
I would say add a link to the company page, it seems that is kept most up to date.

As a side point it might be good to have slack handles on each person on that page btw.